app/scripts/services/membership/roleBindings.js

```diff
@@ -9,26 +9,12 @@ angular
 
     var roleBindingsVersion = APIService.getPreferredVersion('rolebindings');
 
-    var cache = {};
-
-    // recurse until we get a unique binding name compared to what is cached
-    var qualifyBindingName = function(name, suffix) {
-      var search = suffix ?
-                name + suffix :
-                name;
-      return _.some(cache, _.matchesProperty('metadata.name', search)) ?
-              qualifyBindingName(name, _.uniqueId()) :
-              search;
-    };
-
     var bindingTPL = function(role, projectNamespace) {
-      var roleName = _.get(role, 'metadata.name');
-      var bindingName = roleName ? qualifyBindingName(roleName) : null;
       return {
         kind: 'RoleBinding',
         apiVersion: 'v1',
         metadata: {
-          name: bindingName,
+          generateName: _.get(role, 'metadata.name').concat('-'),
```

Comment:
@spadgett got it here.

Reply:
It's generally better to use `+` rather than `concat()` to concat strings. See

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/concat

If we're going to do this, no reason to use `_.get` because it will fail if undefined anyway. So just

```js
role.metadata.name.concat('-')
```

But I'd still prefer

```js
role.metadata.name + '-'
```

Reply:
yeah think i failed a test too actually, updating.